util/has_lib.js

```diff
@@ -69,13 +69,13 @@ function hasLdconfig () {
  */
 function hasFreetype () {
   try {
-    if (childProcess.execSync('pkg-config cairo --cflags-only-I 2>/dev/null | grep freetype2').length) {
+    if (childProcess.execSync('pkg-config cairo --cflags-only-I 2>/dev/null | ' +
+                              'grep freetype2').length) {
       return true
     }
   } catch (err) {
-    // noop
+    return false
   }
-  return false
```

Comment:
Good catch. Yes, my intention was to do it the same way as `hasLdconfig()` and `hasPkgconfigLib()`, I'll fix it. Also `hasSystemLib()` using `length` too, should I remove it there too?